### ISSUE TITLE
Update name for translated field

### ISSUE THREAD

**Issue by max-si-m** [globalize/globalize#574]
Hello, I have deprecation warning like this:

```
DEPRECATION WARNING: You have defined translated attributes with names that conflict with columns on the model table. Globalize does not support this configuration anymore, remove or rename these columns.
Model name (table name): Institution (institutions)
Attribute name(s): name
 (called from check_columns! at /home/max/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/bundler/gems/globalize-abe7786f4cc7/lib/globalize/active_record/act_macro.rb:54)
```

I use `rails 5.0.0.1` and `globalize (5.1.0 abe7786)`. I want to rename translated column. And I made migration: 

```
class RenameConflictColumns < ActiveRecord::Migration[5.0]
  def up
    rename_column :institution_translations, :name, :translated_name
  end

  def down
    rename_column :institution_translations, :translated_name, :name
  end
end
``` 
And model like this:
```
class Institution < ActiveRecord::Base
  #translates :name
  translates :translated_name
end
```
and try it:

```
irb(main):038:0> i = Institution.where(name: 'some name').first_or_create
  Institution Load (0.8ms)  SELECT  "institutions".* FROM "institutions" WHERE "institutions"."name" = $1 ORDER BY "institutions"."id" ASC LIMIT $2  [["name", "some name"], ["LIMIT", 1]]
   (0.3ms)  BEGIN
  SQL (0.8ms)  INSERT INTO "institutions" ("name", "created_at", "updated_at") VALUES ($1, $2, $3) RETURNING "id"  [["name", "some name"], ["created_at", 2016-11-02 15:39:51 UTC], ["updated_at", 2016-11-02 15:39:51 UTC]]
  SQL (0.8ms)  INSERT INTO "institution_translations" ("institution_id", "locale", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"  [["institution_id", 3], ["locale", "en"], ["created_at", 2016-11-02 15:39:51 UTC], ["updated_at", 2016-11-02 15:39:51 UTC]]
   (3.5ms)  COMMIT
=> #<Institution id: 3, name: "some name", created_at: "2016-11-02 15:39:51", updated_at: "2016-11-02 15:39:51">
irb(main):039:0> i.name
=> "some name"
irb(main):040:0> i.translated_name
=> nil

```

I want to use `translated_name` as `name` and without deprecation error. How can I do it?
Have a nice day!

**Comment by max-si-m** [comment]
I don't want to use `translated_name` anywhere, only for remove depreciation warning. Because the project is too big for refactoring each execution.

**Comment by shioyama** [comment]
Maybe the message is confusing, but the point is that you need to change the column named `name` on  your *model table*, i.e. `institutions.name`, because it conflicts with `institution_translations.name`. You don't need to change anything on the translations table.

**Comment by JIAZHEN** [comment]
@shioyama thanks for the reply. I encountered this confusion a few weeks ago, then finally realised it's to remove `name` field in *model* not the translations.

However, there's an issue where if my model has only one column `name`. Removing that field means I have a table that contains ID only.

**Comment by shioyama** [comment]
@JIAZHEN 

> However, there's an issue where if my model has only one column name. Removing that field means I have a table that contains ID only.

That's fine, but don't you have timestamps on the model table?

**Comment by JIAZHEN** [comment]
> That's fine, but don't you have timestamps on the model table?
True

**Comment by ridiculous** [comment]
Is this still an issue?

**Comment by JIAZHEN** [comment]
Nope. PR #577 looks good to me

**Comment by shioyama** [comment]
Merged #577.
